extensions/positron-assistant/src/tools/projectTreeTool.ts

```diff
@@ -134,31 +157,18 @@ export const ProjectTreeTool = vscode.lm.registerTool<ProjectTreeInput>(Positron
 			resultParts.push(new vscode.LanguageModelTextPart(truncationMessage));
 		}
 
+		// Inform the model if results were excluded
+		if (excludedCount > 0) {
+			const exclusionMessage = `${excludedCount} result${excludedCount === 1 ? ' was' : 's were'} excluded. Set \`skipDefaultExcludes\` to \`true\` to see them.`;
+			resultParts.push(new vscode.LanguageModelTextPart(exclusionMessage));
```

Comment:
another log msg suggestion:

```suggestion
			const exclusionMessage = `${excludedCount} result${excludedCount === 1 ? ' was' : 's were'} excluded. Set \`skipDefaultExcludes\` to \`true\` to include them.`;
			log.debug(`[${PositronAssistantToolName.ProjectTree}] ${exclusionMessage}`);
			resultParts.push(new vscode.LanguageModelTextPart(exclusionMessage));
```